### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by ghost** [davmac314/dinit#73] [Show and tell]
I created a AUR package for `dinit`, if you want to put it on the `README.md`. Well... of course it is not a very good PKGBUILD and it has conflicts with the `filesystem` and `systemd-sysvcompat` packages, so if someone wants to improve or own the package feel free to do it.

https://aur.archlinux.org/packages/dinit

**Comment by davmac314** [comment]
I added a link from the project website (https://davmac.org/projects/dinit/) - I've credited it to you as Relwi, if you want that altered just let me know. Thanks!

**Reply by ghost** [reply]
Hey long time, I took the decision to delete my AUR account, the package still exists but I'm not maintaining it anymore, I'm informing you if you want to delete my credit in your page.

**Reply by mobin-2008** [reply]
Hi.

Thanks for all you did for dinit :) I adopted dinit package in AUR.

I wish you the best.

Regards
